packages/react-form/src/validation/validators.ts

```diff
@@ -14,7 +14,7 @@ export function notEmpty(error: ErrorContent<string>) {
 }
 
 export function notEmptyString(error: ErrorContent<string>) {
-  return validator(predicates.notEmptyString)(error);
+  return validator(predicates.notEmptyString, {skipOnEmpty: false})(error);
```

Comment:
I think this will require updating the `notEmptyString` predicate to chain off of the `notEmpty` predicate in order to protect against nullable inputs.